File: 1344-angle-between-hands-of-a-clock/1344-angle-between-hands-of-a-clock.py
class Solution:
    def angleClock(self, hour: int, minutes: int) -> float:
        hour %= 12

        minAngle = minutes * 6.0
        hourAngle = hour * 30.0 + minutes * 0.5
        angle = abs(hourAngle - minAngle)
        return min(angle, 360 - angle)

File: 1344-angle-between-hands-of-a-clock/1344-angle-between-hands-of-a-clock.java
class Solution {
    public double angleClock(int hour, int minutes) {
        hour %= 12;

        double minAngle = minutes * 6.0;
        double hourAngle = hour * 30.0 + minutes * 0.5;
        double angle = Math.abs(hourAngle - minAngle);
        return Math.min(angle, 360 - angle);
    }
}